cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/MetadataDataset.java

```diff
@@ -53,12 +54,12 @@
     .registerTypeAdapter(Id.NamespacedId.class, new NamespacedIdCodec())
     .create();
 
-  private static final String HISTORY_COLUMN = "h";
-  private static final String VALUE_COLUMN = "v";
-  private static final String TAGS_SEPARATOR = ",";
+  private static final Pattern VALUE_SPLIT_PATTERN = Pattern.compile("\\s*[-_,\\s]\\s*");
```

Comment:
should we add a comment explaining the reasoning behind this pattern and the logic of splitting?